api/v1beta1/flinkcluster_types.go

```diff
@@ -411,6 +411,10 @@ type FlinkClusterSpec struct {
 	// Flink image spec for the cluster's components.
 	Image ImageSpec `json:"image"`
 
+	// Specifies serviceAccount name assigned to pods
```

Comment:
"The service account assigned to JobManager, TaskManager and Job submitter Pods. If empty, the default service account in the namespace will be used".

Reply:
fixed